introduce mock server (#125)
OpenCovenant/quill: README.md

Run `npm install` to install the packages that this project requires.

## Running `quill`

Choose in which manner you'd like to interact with quill through the following instructions. The app will automatically
reload if you change any of the source files.

### For Development And Cypress Testing With A Mock Server

> Run `npm run start:mock` to start a dev server at `http://localhost:4200/` and then run `npm run start:mock-server` for

Reviewer: ```suggestion
Run `npm run start:mock` to start **quill** on a dev server at `http://localhost:4200/` and then run `npm run start:mock-server` to start 
```

> a mock server in NodeJS. More information on the latter can be found in its [README](mock-server/README.md).

Reviewer: ```suggestion
a mock server of **ink** in a NodeJS environment. More information on the latter can be found in its [README](mock-server/README.md).
```

### For Development And Testing With An Actual Local `ink` Server

> Run `npm run start` to start a dev server at `http://localhost:4200/` and then follow the instructions on the README

Reviewer: ```suggestion
Run `npm run start` to start **quill** on a dev server at `http://localhost:4200/` and then follow the instructions on the README
```
